CONTRIBUTING.md

```diff
@@ -56,6 +56,12 @@ For more information, see the [DocFX Cross Reference](https://dotnet.github.io/d
 
 ## Images and screenshots
 
+Don't include images with articles, except:
+* In basic onboarding (beginner) tutorials.
+* When an image is needed for clarity.
```

Comment:
I like "needed" better here ... what do you mean by "Personification?"

Reply:
"needed" was&dagger; considered to be something associated with a biological entity. Inanimate objects "require" things and people "need" things. It's very, very old skool. Newspaper editors back in the day would flag it. Here, the document "requires" the image to make it clear to the reader, so they would say that "needs" is inappropriate.

needed&dagger; I'm just about old as dirt and sometimes shocked by things I see in recent AP (or Chicago) style manual guidelines just because I'm soooooo 👴 lol